starter-code/src/app/app.component.html

```diff
@@ -1,3 +1,15 @@
-<h1>
-  {{title}}
-</h1>
+<header id="site-header" >
+  <div class="container">
+    <nav>
+      <a [routerLink]="['/home']">Home</a>
+      <a [routerLink]="['/movie/:id']">Movie</a>
```

Comment:
This second link here would not work. None of your routes will match this. :id is a placeholder only